cpp/src/transform/transform.cpp

```diff
@@ -189,6 +190,7 @@ std::unique_ptr<column> transform(std::vector<column_view> const& inputs,
                                   rmm::cuda_stream_view stream,
                                   rmm::device_async_resource_ref mr)
 {
+  CUDF_EXPECTS(is_runtime_jit_supported(), "Runtime JIT is only supported on CUDA Runtime 11.5+");
```

Comment:
Does this break support for existing cudf features in CUDA < 11.5?

Reply:
No, transforms aren't used internally presently so I don't think it breaks anything.